baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -505,6 +505,72 @@ def mutate_and_get_payload(cls, root, info, **input):
         )
 
 
+class ChatRoomDeleteMessage(RelayMutation):
+    deleted_message = graphene.Field(MessageObjectType._meta.connection.Edge)
```

Comment:
There is no need to return the deleted_message, you are required to return only its relay ID to make relay in the frontend remove the message from the store. This follows better the relay/graphql pattern.